### DISCUSSION TITLE
XHP70.2 led board size

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#7] [Q&A] [answered]
Hi Serhiy,

Is the led pcb size critical?
 
I see there are 16mm or 20mm sizes on aliexpress, etc.

**Comment by srozum** [comment, marked as answer]
Good heat dissipation affects light output stability. I haven't tested LED on 16mm board, so it's up to you.

**Comment by polygonfuture** [comment]
Ok great.  I figured 20mm was the proper size since it has larger heat sink surface but wasn’t sure.  Should we add that detail to the bom to reduce any confusion for others?
